### ISSUE TITLE
Cannot tab to Toggles in Safari

### ISSUE THREAD

**Issue by dabrad26** [carbon-design-system/carbon-website#1944]
When trying to tab through a form you cannot tab to (put focus on) Toggle elements in Safari.  You can in Firefox and Chrome without issue. But Safari is unable to get focus via tab (or even click). Making it unusable without a mouse.

NOTE: this is possible in Safari if you turn on tabbing to all elements (or use OPTIONS + TAB).  However, being a Form element (and for us mostly used in standard Forms) it should work for all users with default systems and setup.

## Environment

> Operating system

Mac OS X

> Browser

Safari 14.0 (15610.1.28.1.9, 15610)

> What version of the Carbon Design System are you using?

    "carbon-components": "^10.22.0",
    "carbon-components-react": "^7.22.0",

> What did you expect to happen?

In standard browser (clean install no preference changes) the tab should go to every form element.

![example2](https://user-images.githubusercontent.com/8028956/97922594-b9ebc380-1d2a-11eb-810f-84945a846789.gif)

We know some elements are not selectable via Safari's odd Tab choices but being a form element I feel this is a bit more critical (and it makes Toggle not a great choice and forces us to recommend Design to always use Checkboxes).

**Comment by emyarod** [comment]
can you create a [reduced test case in Code Sandbox](https://codesandbox.io/s/github/carbon-design-system/carbon/tree/master/packages/react/examples/codesandbox) for easier testing and debugging? when I tab through the Form storybook example (https://carbon-components-react.netlify.app/?path=/story/form--default) I am able to tab to Toggles (even though I have the Safari tab options off)

![Kapture 2020-11-09 at 9 14 18](https://user-images.githubusercontent.com/8265238/98559529-59014580-226c-11eb-9e0e-2bdd6d88174e.gif)

**Comment by dabrad26** [comment]
I just used that link and am unable to Tab to the toggle. I can also not Tab to the checkbox either in latest Safari (I thought I could in the past though)

Safari Version 14.0 (15610.1.28.1.9, 15610)

I am NOT using OPTION-TAB and the setting in Preference is not enabled.

<img width="534" alt="Screen Shot 2020-11-09 at 11 04 57 AM" src="https://user-images.githubusercontent.com/8028956/98565318-78ec3580-227b-11eb-80d3-38bc62386e54.png">

**Comment by dabrad26** [comment]
![example2](https://user-images.githubusercontent.com/8028956/98565515-b224a580-227b-11eb-9e24-884dbfc82527.gif)

**Comment by dabrad26** [comment]
Testing other sites. Seems Safari does not Tab to Radio or Checkboxes at all. So I guess Toggle not Tabbing makes sense following this.  Seems Safari 10 or so added the required checkbox. So I guess will close this. And let design know they may need to work with microcopy to document this accessibility thing in the docs.

**Comment by emyarod** [comment]
yeah, as I mentioned I was not using that a11y option either in Version 14.0 (15610.1.28.1.9, 15610)

![image](https://user-images.githubusercontent.com/8265238/98565834-b056e400-2273-11eb-8bcf-e01100f397c5.png)

that being said I believe this is a known issue with Safari (ignoring tabindex values for certain elements)

**Comment by dabrad26** [comment]
Yeah but you must have something forcing it to tab to them. I checked another machine and default does not tab to them. So you must have some setting causing that to get tabs.  Per Apple it will not be default tab to anything other than text inputs, select.  

Maybe you have an accessibility setting in Mac OS causing it to tab.  

But I did spin up a VM with clean setup and default install does not tab to any of those.
